f/symfony/index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+use App\Kernel;
+use Bref\Bridge\Symfony\SymfonyAdapter;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\HttpFoundation\Request;
+
+ini_set('display_errors', '1');
+error_reporting(E_ALL);
+
+require __DIR__.'/vendor/autoload.php';
+
+Debug::enable();
```

Comment:
Does it need to be debug? Or is the env used here?

Reply:
So [this is something I've wondered about](https://stackoverflow.com/questions/37431935/is-it-normal-to-have-symfony-component-debug-exception-fatalerrorexception-excep), basically it turns errors/warnings into exceptions. For the benchmarks at least it makes sense, and I tend to always use it so that it crashes loudly and I'm aware of it.

I don't think (but I may be wrong) that it affects performances, it simply registers error handlers.